### ISSUE TITLE
Rapid fire is broken i think

### ISSUE THREAD

**Issue by Bunjolio** [Reubinator/PCP-1v1lol-#1]
it jus dont work

**Comment by Reubinator** [comment]
Sorry to hear, scripts working fine since i last checked however they did an update to the map. Maybe it needs to be remade? i will check for you.

**Comment by Reubinator** [comment]
> it jus dont work

Yep one hundred percent right bro, been patched and code needs to be updated, i will fix it soon but im working on a better cheat for 1v1lol with god mode stuff like that, so that's my current priority.

**Comment by Bunjolio** [comment]
> 

alr, lmk if u need testers

**Comment by Reubinator** [comment]
> > 
> 
> alr, lmk if u need testers

Will do, message my discord Reubinator#1769
